ddf-project/ddf-jdbc
# [DDF-JDBC] file system support: s3/hdfs, file format support: avro/orc

This is to sync with the file support pr for ddf core, https://github.com/ddf-project/DDF/pull/285
Reviewers: @hai-adatao 

How to test

Unit test.

PR Progress

Make sure all checkboxes below are checked before merged

 Branch is in format prefix/description (e.g: fix/cast-type)
 Pull Request name is in format [x-xxxx] Add rename API (or [No JIRA])
 Code review is done. Merge check has no conflicts. PR checks passed
